lms/djangoapps/instructor/management/commands/dump_grades.py

```diff
@@ -1,76 +0,0 @@
-#!/usr/bin/python
-"""
```

Comment:
Right, for background, in those cases it was just the browser that was timing out, since grading usually takes more than 60 seconds. In older times, when grading would timeout on the browser, devops would run this management command to get a csv of grades to instructors. Now that grading is a background task, I doubt we have need of this command anymore, but just want to make sure with them.